Library/Homebrew/dependency.rb

```diff
@@ -168,6 +168,16 @@ def merge_temporality(deps)
         [] # Means both build and runtime dependency.
       end
     end
+
+    def expand_dependency(dependency)
```

Comment:
We are referencing it in the method or at least we should in the `expand` that goes later in this method.